projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -44,7 +44,7 @@ <h2 class="cx-product-name">
 
     <div class="row">
       <div class="col-12 col-md-7 col-xl-7">
-        <p class="cx-product-summary" [innerHtml]="product.summary">
+        <p class="cx-product-summary" [innerHTML]="product.summary">
```

Comment:
just curious: do you think that replacing `innerHtml` to `innerHTML` (uppercase HTML) fixed the hydration error? or flickering? (I don't remember the issue we had here)